test/validation/api/timer/timer.c

```diff
@@ -1218,8 +1282,26 @@ static int worker_entrypoint(void *arg ODP_UNUSED)
 	odp_pool_t tbp = global_mem->tbp;
 	uint32_t num_timers = global_mem->timers_per_thread;
 	uint64_t min_tmo = global_mem->param.min_tmo;
+	odp_queue_param_t queue_param;
+	odp_queue_type_t queue_type = ODP_QUEUE_TYPE_PLAIN;
+	odp_thrmask_t thr_mask;
+	struct thread_args *thr_args = arg;
+	uint64_t sched_tmo;
 
-	queue = odp_queue_create("timer_queue", NULL);
+	odp_queue_param_init(&queue_param);
+	if (thr_args->queue_type == ODP_QUEUE_TYPE_PLAIN) {
+		queue_param.type = ODP_QUEUE_TYPE_PLAIN;
+		queue_type = ODP_QUEUE_TYPE_PLAIN;
+	} else {
+		odp_thrmask_set(&thr_mask, odp_thread_id());
```

Comment:
Thread mask needs to be init to zero (odp_thrmask_zero(&thr_mask)) before adding the thread id.